packages/turbo-types/package.json

```diff
@@ -18,12 +18,14 @@
     "build": "tsc && pnpm generate-schema",
     "lint": "eslint src/",
     "lint:prettier": "prettier -c . --cache",
-    "generate-schema": "node scripts/generate-schema.mjs"
+    "generate-schema": "tsx scripts/generate-schema.ts"
```

Comment:
❤️ long live tsx

Reply:
haha I wasn't sure how that'd go over, so this is useful feedback! haha.  I'm pretty lost without types!